android/app/src/main/java/app/organicmaps/car/renderer/Renderer.java

```diff
@@ -125,4 +125,17 @@ public void onZoomOut()
   {
     Map.zoomOut();
   }
+
+  /**
+   * Updates speed limit view.
+   *
+   * @param speedLimit         The speed limit value in formatted units. A value <= 0 will hide the view.
```

Comment:
How would "unlimited" speed limit display on German autobahns? Does it make sense to use 0 or something like 999 for that specific case?

Reply:
Is it feasible to test?